A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js

```diff
@@ -134,6 +134,7 @@ class A320_Neo_PFD_MainPage extends NavSystemPage {
 
         const ADIRSState = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum");
         const PFDAlignedFirst = SimVar.GetSimVarValue("L:A32NX_ADIRS_PFD_ALIGNED_FIRST", "Bool");
+        const PFDAlignedATT = SimVar.GetSimVarValue("L:A32NX_ADIRS_PFD_ALIGNED_ATT", "Bool");
```

Comment:
this line got accidentally deleted in https://github.com/flybywiresim/a32nx/pull/520/commits/7cee13dbe6d426e010698ecfa21c67913d97888f